sdcm/cluster.py

```diff
@@ -420,6 +420,11 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
             DatabaseLogEvent(type='BROKEN_PIPE', severity=Severity.WARNING,
                              regex='cql_server - exception while processing connection:.*Broken pipe'),
             DatabaseLogEvent(type='SEMAPHORE_TIME_OUT', regex='semaphore_timed_out', severity=Severity.WARNING),
+            DatabaseLogEvent(
+                type='DATABASE_ERROR',
```

Comment:
Please, change the type to some unique string, e.g., `EMPTY_NESTED_EXCEPTION` or something like this.